app/scenes/Document/components/Editor.tsx

```diff
@@ -103,9 +103,14 @@ function DocumentEditor(props: Props, ref: React.RefObject<any>) {
 
   React.useEffect(() => {
     if (focusedComment) {
+      history.replace({
```

Comment:
I think this will result in a double render when focusing a comment always. We should check if we're already on the correct route before calling `replace`?

Reply:
Ah yes.. It affects all types of comment focus (I was just thinking of the email flow)..